src/features/PeopleList/styled.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+export const Content = styled.section`
+    max-width: 1368px;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Title = styled.h1`
+    font-size: 36px;
+    line-height: 120%;
+    margin: 0;
+    padding: 56px 16px 24px 16px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        font-size: 20px;
+        padding: 24px 16px 12px 16px;
```

Comment:
Same thing, you could remove last 16px. If you decide to remove padding and set an exact margin above dont forget to do it also here.